src/cmakeProject.ts

```diff
@@ -1493,8 +1493,8 @@ export class CMakeProject {
     }
 
     // Reconfigure if the saved file is a cmake file.
-    async doCMakeFileSaveReconfigure(textDocument: vscode.TextDocument) {
-        const filePath = util.platformNormalizePath(textDocument.uri.fsPath);
+    async doCMakeFileChangeReconfigure(textDocument: vscode.Uri) {
```

Comment:
Changing the type of the argument seems unncessary. Please revert to `vscode.TextDocument`.

Reply:
This is related to checking if a renamed file is a "CMakeLists.txt" file. in "onDidRenameFiles" we had access to URI.